examples/collision/square-platform.cpp

```diff
@@ -1,39 +1,36 @@
-#include <iostream>
+#include <SDL2/SDL_events.h>
+#include <SDL2/SDL_keycode.h>
+#include <SDL2/SDL_rect.h>
+#include <SDL2/SDL_render.h>
 #include <cmath>
-#include <string>
 #include <deque>
-#include <vector>
-#include <chrono>
+#include <iostream>
 #include <map>
+#include <string>
+#include <vector>
 
 #include <SDL2/SDL.h>
 #include <SDL2/SDL2_gfxPrimitives.h>
 
-#include "math/Vector2.hpp"
-#include "collider/ColliderRect.hpp"
+#include "Game.hpp"
+#include "InputHandler.hpp"
+#include "collider/Collider.hpp"
 #include "collider/ColliderCircle.hpp"
+#include "collider/ColliderRect.hpp"
 #include "collider/ColliderScreen.hpp"
-#include "InputHandler.hpp"
-#include "Game.hpp"
+#include "math/Vector2.hpp"
 
 #define SCREEN_WIDTH 800
 #define SCREEN_HEIGHT 600
 #define G 9.8
 
 using namespace std;
 
-
 // keys to InputHandler observe
-std::map<const char*, SDL_Keycode > input_config = {
-    {"quit", SDLK_ESCAPE},
-    {"up", SDLK_UP},
-    {"left", SDLK_LEFT},
-    {"right", SDLK_RIGHT},
-    {"down", SDLK_DOWN},
-    {"velocity_up", SDLK_w},
-    {"velocity_down", SDLK_s},
-    {"run", SDLK_LSHIFT}
-};
+std::map<const char*, SDL_Keycode> input_config = {
```

Comment:
I only have concerns about this. I haven't seen how this displays on my local editor but on Github it looks weird.

Reply:
fixed @rafaelcn